### ISSUE TITLE
Response configuration

### ISSUE THREAD

**Issue by Mattzr** [apigee-127/bagpipes#2]
Hi,

First of all thanks for the great job on swagger-pipes. It is very friendly to used and make apigee 127 even more pleasant.

Then - I have a small concern regarding one of my fitting. Below the very simple code:

module.exports = function create(pipeDef) {

```
return function searchRetailTransactions(context, cb) {

            return cb(null , JSON.stringify({test: "hey"}));
        }
}
```

}

this return me a a plain text but I would like it to be json so I added:

context.headers["Content-Type"] = "application/json"

before the callback but it still doesn't work.

This is how is it setup in my swagger doc:

x-swagger-pipes:

  searchRetailTransaction:
    name: searchRetailTransactions
    type: user

  searchTransactionFlow:
- searchRetailTransaction
  
  Could you help me with that ?

Thanks

**Comment by Mattzr** [comment]
I had to do this to make it work which is not recommended by the documentation:
context.response.setHeader("Content-Type","application/json");

**Comment by theganyo** [comment]
Hey @Mattzr. So glad you like what I've done... it just so happens I'm back looking at it again right now and have a couple changes I just released in 0.1.0. 

First, your original code:

```
context.headers["Content-Type"] = "application/json"
```

should work now. 

Second, you'll have to change your initialization a little bit as `pipesConfig` now expects arrays of strings instead of just strings for all directories:

```
  var pipesConfig = {
    userFittingsDirs: [ path.resolve(__dirname, 'api/fittings') ],
    userViewsDirs: [ path.resolve(__dirname, 'api/fittings') ],
    userControllersDirs: [ path.resolve(__dirname, 'api/controllers' ]
  };
  return swaggerPipes.create(pipesDefs, pipesConfig).connectMiddleware();
```

I'll be marking more changes soon, but hopefully everything will just magically get better for you. :) Please let me know how it goes!

Scott

**Comment by Mattzr** [comment]
It works now .. Awesome ! Thanks for your promptness ! I may come up with few other questions though :)
